### DISCUSSION TITLE
Trying to create a relationship between 2 tables and it is not working

### DISCUSSION THREAD

**Discussion by ramonck** [Budibase/budibase#3482] [Help] [answered]
Hi,

Trying to create a relationship between 2 tables and it is not working please help me with any material on how to make relationships work, thank you.

**Comment by gvlx** [comment]
Hi @ramonck,

Is it user-driven? Do you want the user to pick a relation from a form picker field?
Like this: https://docs.budibase.com/guides/master-detail

Or are you looking for something more like this: https://docs.budibase.com/guides/create-a-child-record-from-the-parent

**Comment by mjashanks** [comment, marked as answer]
Also @ramonck - are you using 
- SQL: https://docs.budibase.com/quickstart-tutorials/crud-app-with-sql/using-relationships-between-sql-tables
- or Budibase tables: https://docs.budibase.com/building-apps/data/data-types/relationships

**Comment by ramonck** [comment]
Guys thank you so much, it worked like a charm. I was doing this process but it was not working I think I was trying to do it in the wrong table the relationship now it worked fine.

Have a great thanksgiving.
